joomla/joomla-cms
# Fix users manage filters language strings

Pull Request for New Issue.
#### Summary of Changes

This PR fixes a inconsistency in com_users filters.

All other filters across joomla admin panel have this format `- Select xxxxxxxxxx -`.

But "Users" -> "Manage" have this

![image](https://cloud.githubusercontent.com/assets/9630530/14743625/51bdb86c-089a-11e6-8ba3-162792ec98fb.png)

this PR fixes that.
#### Testing Instructions

Very simple test.
1. Apply patch
2. Go to "Users" -> "Manage" and click "Search Filters". Check all are as described.

Or just code review.
